Formula/json-fortran.rb

```diff
@@ -1,8 +1,8 @@
 class JsonFortran < Formula
   desc "Fortran 2008 JSON API"
   homepage "https://github.com/jacobwilliams/json-fortran"
-  url "https://github.com/jacobwilliams/json-fortran/archive/4.3.0.tar.gz"
-  sha256 "c97f8de53e2ca9ee91fb3148bfa971b00eaea6e757b5e3d67cc4b4ff197e392e"
+  url "https://github.com/jacobwilliams/json-fortran/archive/5.0.0.tar.gz"
+  sha256 "0a1bf8788bdf8bdc6af009ae078ed374b2c95e0cfc3f30354b5dc5d3d35e5e66"
```

Comment:
JSON-Fortran 4.3 triggered a bug in GCC 6.1 (gfortran) that was not triggered in 5.3. A fix to this bug was included in the latest 5.x release.

We are considering back porting to a 4.4 release since 5.0.0 is not backwards compatible, but provides many improvements and bug fixes